packages/gatsby-theme-carbon/src/components/ExpressiveList/ExpressiveList.module.scss

```diff
@@ -0,0 +1,47 @@
+.listRow {
+  padding: 1rem;
+  padding-bottom: 4rem;
+  border-bottom: 1px solid $ui-03;
+}
+
+:global(.expressive-content-list-group) > .listRow:not(:first-child) {
+  margin-top: 0;
+}
+
+.background {
+  background: $ui-01;
+}
+
+.title {
+  @include carbon--type-style('expressive-heading-04', true);
+
+  padding-right: 2rem;
+
+  @media (max-width: 847px) {
```

Comment:
curious what this is for?

Reply:
I was trying to adjust title with the spacing at the breakpoints. At that point its wraps the title. But ill probably just do below at breakpoint lg